web/main/views.py

```diff
@@ -1731,7 +1731,7 @@ def edit_casebook(request, casebook: Casebook):
     ...    content_includes=[private.title, "You are viewing a private casebook"],
     ... )
     >>> check_response(
-    ...     client.post(private.get_edit_url(), {'title': new_title}, as_user=private.testing_editor),
+    ...     client.post(private.get_edit_url(), {'title': new_title}, as_user=private.testing_editor, follow=True),
```

Comment:
This tells the Django test client to follow the redirect-after-post, so the `check_response` function eventually gets the 200 OK response it's expecting.